packages/chopsticks/src/rpc/substrate/chain.ts

```diff
@@ -1,7 +1,12 @@
 import { Handlers, ResponseError } from '../shared'
 
 const processHeader = (header: any) => {
-  const res = header.toJSON() as any
```

Comment:
this is not a proper fix. Header should have `toJSON` and be typed Header. I think the problem is where block is created, header is not encoded. One place is loadBlocksFromDb. It needs to encode header `registry.createType('Header', header)`

Reply:
```
export const BlockEntity = new EntitySchema<{
  hash: HexString
  number: number
  header: Header /// this is wrong, this is object
  parentHash: HexString | null
  extrinsics: HexString[]
  storageDiff: Record<HexString, HexString | null> | null
}>({
```

Reply:
header type is wrong, needs to be object